be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/DbActor.scala

```diff
@@ -93,7 +118,7 @@ object DbActor extends AskPatternConstants {
   /** Response for a [[Write]] db request Receiving [[DbActorWriteAck]] works as
     * an acknowledgement that the write request was successful
     */
-  final case object DbActorWriteAck extends DbActorMessage
+  final case class DbActorWriteAck() extends DbActorMessage
```

Comment:
Any particular reason why you switched from object to class

Reply:
There actually was, as I had some trouble sending one somewhere in DbActor, which was weird, but would it be better to change it back or is it fine if we keep it as a class?